src/Abp/Runtime/Caching/TypedCacheWrapper.cs

```diff
@@ -56,42 +56,55 @@ public Task ClearAsync()
 
         public TValue Get(TKey key, Func<TKey, TValue> factory)
         {
-            return InternalCache.Get(key, factory);
+            return (TValue)InternalCache.Get(key.ToString(), (k) => factory(key));
         }
 
-        public TValue[] Get(TKey[] keys, Func<TKey, TValue> factory)
+        public async Task<TValue> GetAsync(TKey key, Func<TKey, Task<TValue>> factory)
```

Comment:
Please leave the existing order of grouping overrides, rather than sync + async methods.